data/js/configuring-data.md

```diff
@@ -0,0 +1,164 @@
+# Configuring Data
+
+###### The `api.json` and `api-*.json` files allow us to configure the accessible paths for each data service.
+
+<!-- <article id="undestanding-configuration-files"> -->
+
+## Understanding configuration files
+
+By default WeDeploy Data service is going to use all the JSON files starting with `api-*` and also the file `api.json`.
+
+These files are used to help you manage features such as path validation, authentication, and params validation.
+
+The api JSON files are located at the same path of the `./container.json` and are used following the ordering filesystem.
+
+By the default, if you don't add any api.json file all the collections are going to be exposed.
```

Comment:
Don't we need a folder with a container.json to have data running? And is the `api.json` inside the same folder?